packages/website/content/integrations/dnd-kit.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: 'Dnd kit'
+slug: '/integrations/dnd-kit/'
+section: 'integrations'
+---
+
+We recommend using the excellent [dnd kit](https://dndkit.com/) library for adding drag and drop functionality to Forma 36 components.
+
+## Examples
+
+### Integrating with `useSortable`
```

Comment:
Let's provide one more example with keyboards controls enabled. So with the usage of:
```
useSensors(
    useSensor(PointerSensor),
    useSensor(KeyboardSensor, {
      coordinateGetter: sortableKeyboardCoordinates,
    })
  );
```

And I would also maybe add it to the example with Card component.
WDYT?

Reply:
And might also be valuable to add an example with different UI components from Forma. For instance `Table`. Basically to show that DnD is working with any UI component.

Reply:
The defaults seem to work with keyboard navigation for me, although I'm guessing the KeyboardSensor makes it nicer somehow? I would like to keep the examples as simple as possible, readers can always dig into dnd kit docs if they care for more.